en/SettingsView/General.md

```diff
@@ -119,14 +139,61 @@ Use the *NMEA GPS Device* drop-down selector to manually select the GPS device a
   - **NMEA Stream UDP Port**: The UDP port on which QGC will listen for NMEA data (QGC binds the port as a server)
 
 
+
+## RTK GPS {#rtk_gps}
+
+This section allows you to specify the RTK GPS "Survey-in" settings, to save and reuse the result of a Survey-In operation, or to directly enter any other known position for the base station.
+
+![RTK GPS Settings](../../assets/settings/general/rtk_gps.jpg)
+
+> **Note** The *Survey-In* process is a startup procedure required by RTK GPS systems to get an accurate estimate of the base station position.
+  The process takes measurements over time, leading to increasing position accuracy.
+  Both of the setting conditions must met for the Survey-in process to complete.
+  For more information see [RTK GPS](https://docs.px4.io/en/advanced_features/rtk-gps.html) (PX4 docs) and [GPS- How it works](http://ardupilot.org/copter/docs/common-gps-how-it-works.html#rtk-corrections) (ArduPilot docs).
+
+<span></span>
+> **Tip** In order to save and reuse a base position (because Survey-In is time consuming!) perform Survey-In once, select *Use Specified Base Position* and press **Save Current Base Position** to copy in the values for the last survey.
+  The values will then persist across QGC reboots until they are changed.
+
+The settings are:
+- Perform Survey-In
+  - **Survey-in accuracy (U-blox only):** The minimum position accuracy for the RTK Survey-In process to complete.
+  - **Minimum observation duration:** The minimum time that will be taken for the RTK Survey-in process.
+- Use Specified Base Position
+  - **Base Position Latitude:** Latitude of fixed RTK base station.
+  - **Base Position Longitude:** Longitude of fixed RTK base station.
+  - **Base Position Alt (WGS94):** Altitude of fixed RTK base station.
+  - **Base Position Accuracy:** Accuracy of base station position information.
+  - **Save Current Base Position** (button): Press to copy settings from the last Survey-In operation to the *Use Specified Base Position* fields above.
+  
+
+
+
+
+## ADBS Server {#adbs_server}
+
+![ADBS_Server Settings](../../assets/settings/general/adbs_server.jpg)
+
+The settings are:
+- **Connect to ADBS SBS server**: TBD
```

Comment:
@DonLakeFlyer Can you tell me a bit more about what this is for? How it works? When it should be set/not set?

Reply:
It works pretty close to how MP works: https://discuss.ardupilot.org/t/ads-b-mission-planner-and-flightaware-ads-b-dongle/21362. Not a super easy thing to set up though.

Reply:
Sorry, that is not precise enough. 

So my assumption is that this allows you to connect to an ADBS server "somewhere" in order to get ADBS information. You select connect, enter the URL/port of the server, and QGC shows you info it has from the server about vehicles.
1. Is that about right?
2. Are there public servers or do people have to set up their own?
3. What does QGC display for each vehicle it detects?
2. What gets displayed by QGC if there is a sever